Source/macOS/LoopbackHTTPServer/OIDLoopbackHTTPServer.h

```diff
@@ -72,6 +72,9 @@ typedef enum {
 - (BOOL)start:(NSError **)error;
 - (BOOL)stop;
 
+- (BOOL)hasIPv4Socket;
```

Comment:
I don't know that there's much gain to splitting out part of `startHTTPListener:` into a separate property. The method is only a few lines as is.